generators/server/templates/src/main/resources/config/application-dev.yml.ejs

```diff
@@ -367,6 +367,8 @@ jhipster:
             host: localhost
             port: 5000
             queue-size: 512
+    audit-events:
```

Comment:
Should be available only if generator.authenticationType === 'oauth2' && ['sql', 'mongodb'].includes(generator.databaseType)